api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        class CartController < ::Spree::Api::V2::BaseController
+          def create
+            spree_authorize! :create, Spree::Order
+
+            order = spree_current_order || dependencies[:create_order].call(user: spree_current_user, store: spree_current_store).value
```

Comment:
@bbonislawski let's always create an order here, so always call

```
dependencies[:create_order].call(user: spree_current_user, store: spree_current_store).value
```

Reply:
@damianlegawiec do you think that it's correct to have multiple unfinished orders? and then always take last order as `current` if number wasn't passed in params?